src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
```

Comment:
https://github.com/mate-academy/jv-oop/blob/master/checklist.md#please-dont-add-redundant-empty-lines-to-your-code